converter/include/k4SimDelphes/DelphesEDM4HepConverter.h

```diff
@@ -91,6 +92,8 @@ class DelphesEDM4HepConverter {
     fillReferenceCollection<Electron>(delphesCollection, branch, "electron");
   }
 
+edm4hep::MCRecoParticleAssociationCollection* createExternalRecoAssociations(std::unordered_map<UInt_t, edm4hep::ConstMCParticle> mc_map);
```

Comment:
```suggestion
edm4hep::MCRecoParticleAssociationCollection* createExternalRecoAssociations(const std::unordered_map<UInt_t, edm4hep::ConstMCParticle>& mc_map);
```
With the `map` things in c++ I am never sure if `const` is possible, but I think we should pass it by reference in any case.